components/skus/browser/rs/lib/Cargo.toml

```diff
@@ -22,7 +22,7 @@ tracing = { version = "0.1", default-features = false }
 bigdecimal = { version = "0.1", features = ["serde"] }
 serde = { version = "1.0", features = ["derive"] }
 chrono = { version = "0.4", default-features = false, features = ["clock", "serde"] }
-uuid = { version = "0.8", features = ["serde"] }
+uuid = { version = "1.3", features = ["v4", "serde"] }
```

Comment:
You also need to update the reference in BUILD.gn to the `v1` copy of the crate.